controllers/api_v1/dummy.go

```diff
@@ -0,0 +1,19 @@
+package api_v1
```

Comment:
Should we remove this controller as we don't need that on production @gutakk, this is something useful on `development` phase... as this PR gonna merge we should remove it.

Another thing, we don't have `dummy_test` so not sure what's the goal to keep it

Reply:
Removed in this [PR](https://github.com/gutakk/go-google-scraper/pull/53)

Reply:
I saw that but why don't we remove in this PR as this PR gonna merge to develop branch before that PR :D

Reply:
That make sense. Removed in bee4297ad3e629a4ba78b8a34cb05d08386b3eda